controllers/comments-controller.js

```diff
@@ -0,0 +1,12 @@
+const {fetchCommentsByreviewID} = require("../models/comments-model")
+
+
+exports.getCommentsByReviewID = (req, res, next) => {
+    const {review_id} = req.params;
+    fetchCommentsByreviewID(review_id).then((comments) => {
```

Comment:
you already have a function written that you can use to check whether a review exists by id. 
you can run that first to handle errors, not even use its return value, and then invoke this getComments... function in the .then block

Reply:
okay thank you